conference.js

```diff
@@ -1330,6 +1330,8 @@ export default {
      * @returns {Promise}
      */
     useVideoStream(newStream) {
+        const oldStream = this.localVideo;
```

Comment:
This is a nit so don't take it seriously: any reason why you declared the variable up here instead of closer to where it's being used? Or any reason why not to re-use this variable on line 1336? I wonder if you could even get away with not using an intermediary variable but maybe at the cost of readability.